Cargo.toml

```diff
@@ -35,7 +35,7 @@ hex = "0.4.3"
 jwt-simple = { version = "0.12", default-features = false, features = [
     "pure-rust",
 ] }
-kbs_protocol = { git = "https://github.com/confidential-containers/guest-components.git", rev = "591d0bb", default-features = false }
+kbs_protocol = { git = "https://github.com/mythi/guest-components.git", branch = "nv", default-features = false }
```

Comment:
should we wait for merge nv branch to main ?

Reply:
see https://github.com/confidential-containers/trustee/pull/827#issuecomment-3077211333